chart/gatekeeper-operator/templates/gatekeeper.yaml

```diff
@@ -13,7 +13,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.2.4
+    controller-gen.kubebuilder.io/version: v0.2.2
```

Comment:
should we stick with `0.2.4` or revert back to `0.2.2`

Reply:
Doesn't matter either way as this is informational metadata. We could bump the controller-gen version installed in the Makefile (which is currently v0.2.2)

Reply:
yea let's bump it in Makefile

Reply:
bumped